panic in modd on OSX watching file in SMB volume

As reported in  https://github.com/cortesi/modd/issues/22 I get a panic from the notify code in modd when I watch a file in OSX on a SMB volume (mounted from an NFS server). The same modd never panics when watching files on the local disk of the OSX machine.

OSX 10.10.5.
modd 0.3 built with Go 1.6.

```
09:06:03: prep: make
make: `process.ps' is up to date.
>> done (4.173ms)
panic: unhandled error: notify.FSEventsMustScanSubDirs|notify.FSEventsIsDir|notify.FSEventsInodeMetaMod

goroutine 18 [running]:
panic(0x422e4e0, 0xc82000ab00)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
github.com/cortesi/modd/vendor/github.com/rjeczalik/notify.(*watch).Dispatch(0xc820096580, 0xc820148980, 0x2, 0x2)
	/Users/fcy/go/src/github.com/cortesi/modd/vendor/github.com/rjeczalik/notify/watcher_fsevents.go:136 +0x54f
github.com/cortesi/modd/vendor/github.com/rjeczalik/notify.(*watch).Dispatch-fm(0xc820148980, 0x2, 0x2)
	/Users/fcy/go/src/github.com/cortesi/modd/vendor/github.com/rjeczalik/notify/watcher_fsevents.go:205 +0x3e
github.com/cortesi/modd/vendor/github.com/rjeczalik/notify.gostream(0x4b00ba0, 0x1, 0x2, 0x4b00d90, 0x47ff000, 0x47fe000)
	/Users/fcy/go/src/github.com/cortesi/modd/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:104 +0x11c
github.com/cortesi/modd/vendor/github.com/rjeczalik/notify._cgoexpwrap_b138320e5461_gostream(0x4b00ba0, 0x1, 0x2, 0x4b00d90, 0x47ff000, 0x47fe000)
	??:0 +0x53
github.com/cortesi/modd/vendor/github.com/rjeczalik/notify._Cfunc_CFRunLoopRun()
	??:0 +0x36
github.com/cortesi/modd/vendor/github.com/rjeczalik/notify.init.1.func1()
	/Users/fcy/go/src/github.com/cortesi/modd/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:68 +0x112
created by github.com/cortesi/modd/vendor/github.com/rjeczalik/notify.init.1
	/Users/fcy/go/src/github.com/cortesi/modd/vendor/github.com/rjeczalik/notify/watcher_fsevents_cgo.go:70 +0x48
```

--- a/watcher_fsevents.go
+++ b/watcher_fsevents.go
@@ -133,7 +133,7 @@ func (w *watch) Dispatch(ev []FSEvent) {
 			ev[i].Flags, ev[i].Path, i, ev[i].ID, len(ev))
 		if ev[i].Flags&failure != 0 {
 			// TODO(rjeczalik): missing error handling
-			panic("unhandled error: " + Event(ev[i].Flags).String())
+			continue
 		}
 		if !strings.HasPrefix(ev[i].Path, w.path) {
 			continue

